frontend/views/containers/dashboard/GroupMembersAllModal.vue

```diff
@@ -1,18 +1,254 @@
+
+        i18n.c-member-count.has-text-1(
+          v-if='form.search && searchCount === 0'
+          tag='div'
+          :args='{searchTerm: `<strong>${form.search}</strong>`}'
+        ) Sorry, we couldn't find anyone called "{searchTerm}"
+
+        i18n.c-member-count.has-text-1(
+          v-else
+          tag='div'
+          :args='{ groupMembersCount }'
+        ) {groupMembersCount} members
+
+      transition-group.c-group-list(
+        v-if='searchResult'
+        name='slide-list'
+        tag='ul'
+      )
+        li.c-group-member(
+          v-for='{username, displayName} in searchResult'
+          :key='username'
+        )
+          .c-identity
+            user-image(:username='username')
+            .c-name(data-test='username')
+              strong {{ displayName ? displayName : username }}
+              .c-display-name(
+                data-test='profileName'
+                v-if='displayName'
+              ) @{{ username }}
+
+          .c-actions
+            group-member-menu.c-action-menu
+
+            .c-actions-buttons.buttons
+              button.button.is-outlined.is-small(
+                @click='toChat'
+              )
+                i.icon-comment
+                i18n Send message
+              button.button.is-outlined.is-small(
+                @click='openModal("RemoveMember")'
+              )
+                i.icon-times
+                i18n Remove member
 </template>
 
 <script>
+import sbp from '~/shared/sbp.js'
+import { mapGetters } from 'vuex'
+import { OPEN_MODAL } from '@utils/events.js'
 import ModalBaseTemplate from '@components/modal/ModalBaseTemplate.vue'
+import UserImage from '@components/UserImage.vue'
+import GroupMemberMenu from '@containers/dashboard/GroupMemberMenu.vue'
 
 export default {
   name: 'GroupMembersAllModal',
   components: {
-    ModalBaseTemplate
+    ModalBaseTemplate,
+    UserImage,
+    GroupMemberMenu
   },
-  props: {
-    name: String
+  data () {
+    return {
+      form: {
+        search: null
+      }
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'groupProfiles',
+      'globalProfile',
+      'groupMembersCount'
+    ]),
+    searchResult () {
+      return Object.keys(this.groupProfiles)
+        .map(username => {
+          const inList = (n) => n.toUpperCase().indexOf(this.form.search.toUpperCase()) > -1
+          const { displayName } = this.globalProfile(username)
+          if (!this.form.search || inList(username) || (displayName ? inList(displayName) : false)) {
+            return { username, displayName }
+          }
+        })
+        .filter(profile => profile !== undefined)
+    },
+    searchCount () {
+      return Object.keys(this.searchResult).length
+    }
+  },
+  methods: {
+    openModal (name) {
+      sbp('okTurtles.events/emit', OPEN_MODAL, name)
+    },
+    closeModal () {
+      this.$refs.modal.close()
+    },
+    toChat () {
+      this.$router.push({ path: '/chat' })
```

Comment:
Same issue [as above](https://github.com/okTurtles/group-income-simple/pull/818/files#r376741116), I don't think `/chat` is a valid route, but `/messages` seems to be? Dunno, that stuff is also not implemented, so it's up to you whether you want to change it or leave it as-is for now.